lib/markdownlint/frontmatter.js

```diff
@@ -0,0 +1,47 @@
+const yaml = require('js-yaml')
+
+module.exports = {
+  names: ['front-matter'],
+  description: 'YAML frontmatter',
+  tags: ['local'],
+  function: (params, onError) => {
+    const { config, frontMatterLines } = params
+    const { required_keys: requiredKeys = [] } = config
```

Comment:
is this pulled from `.markdownlint.yml`? and do we need to write an explicit linting function checking for the title?